operating/create-use-collections.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: ASSEMBLY
+[id="create-use-collections"]
+= Creating and using object collections
+include::modules/common-attributes.adoc[]
+:context: create-use-collections
+
+toc::[]
+
+[role="_abstract"]
+{product-title} provides a method for grouping parts of your infrastructure into collections. Collections are similar to Kubernetes scopes and allow you to group elements together for exclusion or inclusion when performing actions in the system; for example, when creating reports to alert you to vulnerabilities in a group of applications in the production system. 
```

Comment:
I like the provided example. When we expand usage of collections to other parts of the product this would be a good place to keep a comprehensive list